sympy/vector/tests/test_printing.py

```diff
@@ -27,20 +29,20 @@ def upretty(expr):
 v.append(N.i)
 v.append(-N.i)
 v.append(N.i + N.j)
-v.append(a*N.i)
-v.append(a*N.i - b*N.j)
-v.append((a**2 + N.x)*N.i + N.k)
-v.append((a**2 + b)*N.i + 3*(C.y - c)*N.k)
+v.append(a * N.i)
```

Comment:
I don't know why, but some expression change their order like was `c - C.y` and know `C.y - c`